app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -115,17 +115,7 @@ const ListingDetail = () => {
                 listing={listing}
                 imageSrc={getAssetPath("listing-legal.svg")}
               />
-              <div className="listing-detail-panel p-0">
-                <div className="m-0 info-card flex items-center justify-between">
-                  <p className="m-0 text-base text-serif-lg w-3/4">
-                    {t("listings.monitoredByMohcd")}
-                  </p>
-                  <img
-                    alt={t("listings.equalHousingOpportunityLogo")}
-                    src={getAssetPath("logo-equal.png")}
-                  />
-                </div>
-              </div>
```

Comment:
so excited the console is going to be so much quieter! thanks for working on this mark!